eslint: Trailing spaces not allowed

Webpack + Eslint compilation shows the error below, because the _lumigo folder wrappers have traling spaces at the end of the files.

Can't use plugin unless I disable eslint error.

error:
```
  8:1  error  Trailing spaces not allowed  no-trailing-spaces
```

example:
![Screenshot from 2020-01-24 20-24-41](https://user-images.githubusercontent.com/7067145/73093634-9a945000-3ee7-11ea-8391-a478011e0f67.png)

--- a/src/index.js
+++ b/src/index.js
@@ -305,8 +305,7 @@ const tracer = require("@lumigo/tracer")({
 });
 const handler = require('../${handlerModulePath}').${handlerFuncName};
 
-module.exports.${handlerFuncName} = tracer.trace(handler);
-    `;
+module.exports.${handlerFuncName} = tracer.trace(handler);`;
 
 		const fileName = localName + ".js";
 		// e.g. hello.world.js -> /Users/username/source/project/_lumigo/hello.world.js

